actions/imanage/functions/imanage-login/1.0/index.js

```diff
@@ -0,0 +1,33 @@
+import urlSearchParams from "../../utils/url-search-params"
+
+const AUTHURL = "https://cloudimanage.com/auth/oauth2/token"
+
+const imanageLogin = async ({email, password, client_id, client_secret}) => {
+    const loginData = new urlSearchParams({
+        username: email,
+        password,
+        client_id,
+        client_secret,
+        grant_type: "password"
+    })
+
+    return await fetch(AUTHURL, {
+        method: "post",
+        body: loginData.toString(),
+        headers: { "Content-Type": "application/x-www-form-urlencoded"}
+    })
+    .then(response => {
+        if (response.status === 200 && response.ok === true) {
+            const tokens = response.json()
+            return {
+                token: tokens.access_token
+            }
+        }
+        throw Error(`${JSON.stringify(response)}`)
```

Comment:
Do we want to throw an error or use console.error() ?

Reply:
Is both logging and throwing Error alright?

Reply:
We can test :)